lib/http/base.js

```diff
@@ -192,6 +192,8 @@ HttpClient.prototype.getAgent = function (url, proxy, mock) {
   return null;
 };
 
+HttpClient.prototype.post = axios.post;
```

Comment:
cannot we here hide all the agent configuration things?

Reply:
it isn't needed now